### PR TITLE
pygtk: add python_version

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

**Comment by Chocimier** [comment]
First pycompile_module is not needed, as it is actually useful.

Second should be replaced with `pycompile_dirs="usr/lib/pygtk"`

**Comment by Chocimier** [comment]
Horrible typo in previous comment.

**Comment by fosslinux** [comment]
I believe xlint is wrong, but i am not sure.

**Comment by Chocimier** [comment]
Yes, xlint is wrong about `pycompile_module="gtk-2.0/gtk"`, it is set here to avoid compiling whole gtk-2.0 that is shared between packages,. Please add comment and ignore warning.
`pycompile_module="demos"` should be removed because demos is not in site-packages/; `pycompile_dirs="usr/lib/pygtk"` should be set here.

File: srcpkgs/pygtk/template [hunk 1]
```diff
@@ -2,20 +2,24 @@
 pkgname=pygtk
 version=2.24.0
 revision=13
-lib32disabled=yes
 build_style=gnu-configure
 configure_args="PYTHON=python2"
+# This is required to avoid compiling all of gtk-2.0
+# Ignore xlint on this one
+pycompile_module="gtk-2.0/gtk"
+pycompile_dirs="usr/lib/pygtk"
```
Comment (Chocimier) [inline srcpkgs/pygtk/template:10]: usr/lib/pygtk is not part of main package, so no need to set here

Comment (fosslinux) [reply srcpkgs/pygtk/template]: Done

**Comment by fosslinux** [comment]
> Please revbump for pycompile_dirs.

Also done

**Pull request merged**